integration/test_aggregation_integration.py

```diff
@@ -1264,3 +1264,51 @@ def test_concurrent_updates_to_kv_table(setup_teardown_test):
     result = controller.await_termination()
 
     assert result == [{'mykey': 'onekey', 'attr1': 9, 'attr2': 9}]
+
+
+def test_separate_aggregate_steps(setup_teardown_test):
+    def map(x):
+        x['some_data'] = x['some_data'] * 10
+        return x
+
+    t0 = pd.Timestamp(test_base_time)
+    data = pd.DataFrame(
+        {
+            'first_name': ['moshe', 'yosi', 'katya'],
+            'some_data': [1, 2, 3],
+            'time': [t0 - pd.Timedelta(minutes=25), t0 - pd.Timedelta(minutes=30),
+                     t0 - pd.Timedelta(minutes=35)]
+        }
+    )
+
+    table = Table(setup_teardown_test, V3ioDriver())
+
+    controller = build_flow([
+        DataframeSource(data, key_field='first_name', time_field='time'),
+        AggregateByKey([FieldAggregator("number_of_stuff", "some_data", ["avg"],
+                                        SlidingWindows(['1h'], '10m'))], table),
+        Map(map),
+        AggregateByKey([FieldAggregator("number_of_stuff2", "some_data", ["sum"],
+                                        SlidingWindows(['2h'], '10m'))], table),
+        NoSqlTarget(table),
+
```

Comment:
Extra blank line here inconsistent with style elsewhere (including in this test).